src/context/Theme.tsx

```diff
@@ -415,6 +415,11 @@ export const dataGridListStyling: SxProps<Theme> = {
     },
 };
 
+export const disabledButtonText = {
+    light: 'rgba(0, 0, 0, 0.26)',
+    dark: 'rgba(255, 255, 255, 0.3)',
+};
```

Comment:
What is driving needing custom disabled styling that isn't applied to all buttons?